t_reconf/test_reconf_base.py

```diff
@@ -139,7 +146,7 @@ def _start_all_services_and_reload_tempesta(self, first_config: str, second_conf
         tempesta.config.set_defconfig(second_config + self.base_tempesta_config)
         tempesta.reload()
 
-    @parameterize.expand([param("http"), param("https"), param("h2")])
+    @parameterize.expand([param("http"), param("https"), param("h2"), param("h2,https")])
```

Comment:
You shouldn't use "h2,https" here because it is suffix of function name.  I checked it and it looks like this comma replaced with an underscore. I received name of function - `test_reconf_proto_h2_https`. I suggest use two params for these tests: `param(name="h2_https", proto="h2,https")`.